src/tasks/make-util.js

```diff
@@ -30,7 +30,7 @@ var cd = function (dir) {
     var cwd = process.cwd();
     if (cwd != dir) {
         console.log('');
-        console.log(`> cd ${path.relative(cwd, dir)}`);
+        console.log(`> cd ${path.relative(cwd, String(dir))}`);
```

Comment:
This was a weird and not very helpful error message, wasn't it?

Reply:
It wasn't too bad. The error was `TypeError [ERR_INVALID_ARG_TYPE]: The "to" argument must be of type string. Received an instance of String`

Reply:
string != String. Gotcha 😄